coda/coda_mdstore/views.py

```diff
@@ -662,14 +662,14 @@ def externalIdentifierSearch(request, identifier=None):
     Return a collection based on an identifier
     """
 
-    if identifier or request.REQUEST.get('ark'):
+    if identifier or request.POST.get('ark'):
         feed_id = request.path
-        if request.REQUEST.get('ark') and ('ark:/%d' % settings.ARK_NAAN) not in \
-                request.REQUEST.get('ark'):
-            identifier = 'ark:/%d/%s' % (settings.ARK_NAAN, request.REQUEST.get('ark'))
+        if request.POST.get('ark') and ('ark:/%d' % settings.ARK_NAAN) not in \
+                request.POST.get('ark'):
+            identifier = 'ark:/%d/%s' % (settings.ARK_NAAN, request.POST.get('ark'))
             feed_id = request.path + identifier + '/'
-        elif request.REQUEST.get('ark'):
-            identifier = request.REQUEST.get('ark')
+        elif request.POST.get('ark'):
+            identifier = request.POST.get('ark')
```

Comment:
It looks to me like we should actually be using `request.GET.get` in these changes, not `request.POST.get`. This is based on:
1. in the test for this view, `request = rf.get(url, {'ark': 'ark:/%d/metadc000001' % (settings.ARK_NAAN,)})` using a `get` method
2. having to change the call in the test to get it to pass with this change
3. Looking at the form at `/extidentifier/` which shows `get` method `<form action="/extidentifier/" method="get">`

Reply:
ok! I'll try using GET instead of POST here and undo the changes in tests and see if it works.